src/vm/excep.cpp

```diff
@@ -7164,6 +7164,13 @@ EXTERN_C void JIT_MemCpy_End();
 
 EXTERN_C void JIT_WriteBarrier_End();
 EXTERN_C void JIT_CheckedWriteBarrier_End();
+#else
+#ifdef FEATURE_PAL
+EXTERN_C void JIT_WriteBarrierGroup();
```

Comment:
`CHECK_RANGE` macro uses `name` and `name##_End`.  To follow this macro `JIT_WriteBarrier` is already used by `UniversalWriteBarrierHelper` and there are a group of methods so used `Group` prefix.